README.md

```diff
@@ -1,30 +1,47 @@
 # gRPC Swift
 
-This repository contains a gRPC code generator and runtime libraries for Swift.
-You can read more about gRPC on the [gRPC project's website][grpcio].
-
-## Versions
-
-gRPC Swift is currently undergoing active development to take full advantage of
-Swift's native concurrency features. The culmination of this work will be a new
-major version, v2.x. Pre-release versions will be available in the near future.
-
-In the meantime, v1.x is available and still supported. You can read more about
-it on the [Swift Package Index][spi-grpc-swift-main].
-
-## Security
-
-Please see [SECURITY.md](SECURITY.md).
-
-## License
-
-gRPC Swift is released under the same license as [gRPC][gh-grpc], repeated in
-[LICENSE](LICENSE).
-
-## Contributing
-
-Please get involved! See our [guidelines for contributing](CONTRIBUTING.md).
+This repository contains a gRPC implementation for Swift. You can read more
+about gRPC on the [gRPC project's website][grpcio].
+
+> gRPC Swift v2.x is under active development on the `main` branch and takes
+> full advantage of Swift's native concurrency features.
+>
+> v1.x is still supported and maintained on the `release/1.x` branch.
+
+- 📚 **Documentation** is available on the [Swift Package Index][spi-grpc-swift]
+- 💻 **Examples** are available in the [Examples](Examples) directory
+- 🚀 **Contributions** are welcome, please see [CONTRIBUTING.md](CONTRIBUTING.md)
+- 🪪 **License** is Apache 2.0, repeated in [LICENSE](License)
+- 🔒 **Security** issues should be reported via the process in [SECURITY.md](SECURITY.md)
+
+## Quick Start
```

Comment:
Should we maybe link to the hello world tutorial from the Quick Start section, maybe below the Package.swift? I'm thinking, someone may come in wanting to play with this, copy the manifest, and then might struggle a bit clicking around documentations on SPI. Having the link to the tutorial directly may make things slightly easier.

Reply:
Oh that's a good point. I'd like the README to remain really focused by just having the bullet points (the manifest is the exception because it's easy to copy and paste). What if we add a "Tutorials" bullet to the list above between Documentation and Examples?